src/sql/src/session/vars.rs

```diff
@@ -760,6 +760,25 @@ static LOGGING_FILTER: Lazy<ServerVar<CloneableEnvFilter>> = Lazy::new(|| Server
     internal: true,
 });
 
+static ENABLE_COORDINATOR_SECRETS_CACHING: ServerVar<bool> = ServerVar {
+    name: UncasedStr::new("enable_coordinator_secrets_caching"),
```

Comment:
Question: are we likely to keep this when we move webhook sources to `clusterd`? If so, would it be better to call this parameter `enable_webhooks_secrets_caching`?

Reply:
Also, do you need this to be a separate parameter from `coordinator_secrets_caching_ttl`? PostgreSQL convention is to use `0` to reflect `off`, and that's also very natural here, as a "0 TTL" has the right semantics (don't cache).